functions/exercise2/script.js

```diff
@@ -1 +1,30 @@
 "use strict";
+
+function isPalindrome(phrease) {
+    var isPhreasePalindrome = phrease.toLowerCase().replace(/[^a-zA-Z0-9]+/g, "");
```

Comment:
This line causes a defect, because "kayakkayak kayak" is palindrome, while it is not a palindrome.